scripts/generate_assets.py

```diff
@@ -0,0 +1,101 @@
+from birdclef.models.embedding.tilenet import TileNet
```

Comment:
I think this script should move into the birdclef.workflows directory. The file could be named `evaluation.py`, with a click group named `evaluation`.